src/component/chatting/inputChat.tsx

```diff
@@ -0,0 +1,106 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { useRecoilState, useRecoilValue } from 'recoil';
+import { chatList, selectedUser } from '../../atom/atom';
+import { ChatInfo, ChattingRoom } from '../../interface/interface';
+
+const InputChat = ({ chattingRoomId }: ChattingRoom) => {
+  const [text, setText] = useState<string>('');
+  const [newChatList, setNewChatList] = useRecoilState(chatList);
+  const [realChat, setRealChat] = useState(newChatList[chattingRoomId].message);
+  const selectedId = useRecoilValue(selectedUser);
+
+  let updateChat;
+
+  const submitText = (e?: React.FormEvent<HTMLFormElement>) => {
+    e?.preventDefault();
+    if (text.trim()) {
+      let newChat: ChatInfo = {
+        userNum: selectedId,
+        addText: text,
+        date: String(new Date()),
```

Comment:
```suggestion
        date: new Date().toString(),
```
이부분은 이렇게 작성하는것이 같은 `String type`으로 변환된다고 하더라도 `Date`객체에 대한 더 많은 정보를 포함할 수 있다고 합니다!

Reply:
와우 이거 완전 좋은 정보네요!! 두 방법의 차이를 몰라서 전자로 했는데.. 알려주셔서 감사합니다!! 굿굿~